rust/agama-cli/src/auth.rs

```diff
@@ -1,34 +1,34 @@
-use clap::{arg, Args, Subcommand};
+use clap::Subcommand;
 
 use reqwest::header::{HeaderMap, HeaderValue, CONTENT_TYPE};
 use std::fs;
 use std::fs::File;
-use std::io;
+use std::io::{self, IsTerminal};
 use std::io::{BufRead, BufReader};
 use std::os::unix::fs::PermissionsExt;
 use std::path::{Path, PathBuf};
 
+use crate::error::CliError;
+
 const DEFAULT_JWT_FILE: &str = ".agama/agama-jwt";
 const DEFAULT_AGAMA_TOKEN_FILE: &str = "/run/agama/token";
 const DEFAULT_AUTH_URL: &str = "http://localhost/api/auth";
 const DEFAULT_FILE_MODE: u32 = 0o600;
 
 #[derive(Subcommand, Debug)]
 pub enum AuthCommands {
-    /// Login with defined server. Result is JWT stored locally and made available to
-    /// further use. Password can be provided by commandline option, from a file or it fallbacks
-    /// into an interactive prompt.
-    Login(LoginArgs),
-    /// Release currently stored JWT
+    /// Authenticate with Agama's server and store the credentials
```

Comment:
I would document that it can read password from stdin or it will ask on tty. with example like `agama auth login < /tmp/root_pwd`